src/main/java/racingGame/RacingGame.java

```diff
@@ -48,4 +47,26 @@ private void saveRound() {
 
         rounds.add(new Round(positions));
     }
+
+    public List<String> winners() {
+        int max = maxPosition();
+        List<String> result = new ArrayList<>();
+
+        for (Car car : cars) {
+            if (car.position() == max) {
+                result.add(car.name());
+            }
+        }
+        return result;
+    }
+
+    private int maxPosition() {
+        int max = 0;
+        for (Car car : cars) {
+            if (car.position() > max) {
+                max = car.position();
+            }
```

Comment:
객체의 값을 꺼내지 말고 메시지를 보내 구현해 본다.
```suggestion
            max = car.max(max);
```